tests/utils/helper.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import subprocess
+
+from ansible.parsing.dataloader import DataLoader
+from ansible.inventory.manager import InventoryManager
+
+
+def run_make_shell(args=None, tmpdir=None, **kwargs):
+    make_args = []
+    if tmpdir:
+        make_args.append('SHELL_ENV={}'.format(tmpdir))
+    if args:
+        command = args
+    else:
+        command = 'true'
+    make_args.append('C="{}"'.format(command))
+    full_command = 'make shell {args}'.format(args=' '.join(make_args))
+    logging.warning("Running: {}".format(full_command))
```

Comment:
Not really a warning :)

Reply:
but debug are not show by default and I can't find a way to change logging level in pytest directly (I put a commented line, in toplevel conftest to change temporarly the logging, level but you disliked it and made me removed). May be a separated PR to work on fine tunning of log-level of each module (and avoiding having the debug of pytest)

Reply:
What about `info`? But fair enough, not a blocker.

Reply:
Info are not displayed either by default. But might be a solution (decrease the debugging level to info, and set it to info, but need more investigation)